.gitignore

```diff
@@ -30,6 +30,9 @@
 .pub-cache/
 .pub/
 /build/
+pubspec.lock
```

Comment:
why do u ignore pubspec.lock twice? here and lower

Reply:
I didn't see the the lower(, ignored